javascript-conditionals/main.js

```diff
@@ -83,31 +86,30 @@ console.log(
 
 function categorizeAcidity(pH) {
   if (pH === 7) return 'neutral';
+  else if (pH < 0) return 'invalid pH level';
   else if (pH < 7) return 'acid';
   else if (pH <= 14) return 'base';
   else return 'invalid pH level';
 }
 
-console.log('Value of categorizeAcidity:', categorizeAcidity(9));
+console.log('Value of categorizeAcidity:', categorizeAcidity(6));
 
 // introduceWarnerBro(name)
 
-function introduceWarnerBro(name) {
+function introduceWarnerBro(name = 'Goodnight everbody!') {
```

Comment:
You shouldn't be setting the value of `name` inside here. You should just be passing in name. Should just be `function introduceWarnerBro(name) {`